### PR TITLE
ENH fix package installs maybe

### LINKED ISSUE #3: install is not working
It appears that the package is putting its libs in `lib/python2.7/` in my python 3.6 environment.

Any ideas here @jseabold?

This is the cause of some trouble [here](https://github.com/civisanalytics/python-glmnet/issues/32).

### PR DESCRIPTION
**Pull request by beckermr**
These changes fixed the package installs locally. I also went ahead and remove the windows stuff that was not being used. It is in the history, so we can always get it back. 

Closes #3

### CODE REVIEW

**Comment by beckermr** [comment]
It looks like the linter is upset. I would be ok requiring a recent version of numpy, like 1.11 or higher.

**Comment by beckermr** [comment]
@jakirkham @jseabold for viz

**Comment by jakirkham** [comment]
Setting `numpy` to 1.11 should be fine. That preserves the `numpy` version this feedstock already supports. Could use older versions if needed (and are ok with some added complexity in the recipe).

**Comment by jakirkham** [comment]
@conda-forge-admin, please rerender.

**Comment by beckermr** [comment]
Travis is crazy backlogged so this is going to be a few days before it passes.

**Comment by beckermr** [comment]
@jseabold or @jakirkham - it passed! Can we merge?

**Comment by jakirkham** [comment]
Seems fine. Will defer to the maintainer though.

**Comment by beckermr** [comment]
Sounds good. @jseabold?

**Comment by beckermr** [comment]
@jseabold? Can we merge this?

File: recipe/build.sh [hunk 1]
```diff
@@ -2,4 +2,4 @@
 
 # see https://github.com/numpy/numpy/issues/7427
 LDFLAGS="$LDFLAGS -undefined dynamic_lookup -bundle"
-$PYTHON -m pip install .
+$PYTHON setup.py install
```
Comment (beckermr) [inline recipe/build.sh:5]: This is the main change @jseabold.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -46,3 +46,4 @@ about:
 extra:
   recipe-maintainers:
     - jseabold
+    - beckermr
```
Comment (beckermr) [inline recipe/meta.yaml:49]: The changes in this file are by me too.

Comment (beckermr) [reply recipe/meta.yaml:49]: The rest of the changes in the PR are due to conda-forge rerendering the feedstock to update the build.

File: recipe/bld.bat [hunk 3]
```diff
@@ -1,2 +0,0 @@
-"%PYTHON%" setup.py install 
-if errorlevel 1 exit 1
```
Comment (beckermr) [inline recipe/bld.bat:2]: I also removed this unused file.

**Comment by jakirkham** [comment]
Sorry progress has been slow here, @beckermr. Let's go ahead and merge. If there are other issues, we can follow-up on them in a subsequent issue/PR as needed. Thanks for the refresh PR. :)

**Pull request merged**